tests/test_suggester.py

```diff
@@ -82,33 +82,19 @@ def test_get_concrete_release(conf, link, concrete):
 @pytest.mark.parametrize(
     ("json_in", "release_id", "mode", "changed", "json_out"),
     [
-        # unstable happy path
+        # testing happy path
         (
             """{"links": {
-"unstable-py27": "1994.12.00.rc0-py27",
+"testing-py27": "1994.12.00.rc0-py27",
```

Comment:
Now the dictionary has two `testing-py27` keys I am not sure if this is expected or not.

Reply:
Nope, not needed. Nice catch.